packages/astro/src/core/render/slots.ts

```diff
@@ -0,0 +1,82 @@
+import type { SSRResult } from '../../@types/astro.js';
```

Comment:
The entity of this change is missing from the description; could you please add it?

Reply:
This is not new. This file is `result.ts` renamed after createResult was removed from it.